birdhouse/scripts/backup-juputerhub-notebooks.sh

```diff
@@ -5,12 +5,7 @@ if [ -z "$BACKUP_OUT_DIR" ]; then
     BACKUP_OUT_DIR=/tmp
 fi
 
-if [ -z "$DOCKER_JUPYTERHUB_USER_PERSISTENCE_VOLUME" ]; then
-    # dupe with pavics-compose.sh
-    DOCKER_JUPYTERHUB_USER_PERSISTENCE_VOLUME=jupyterhub_user_persistence
-fi
-
-./backup-datavolume.sh "$DOCKER_JUPYTERHUB_USER_PERSISTENCE_VOLUME" "$BACKUP_OUT_DIR"
```

Comment:
Don't call `backup-datavolume.sh`, adapt this script for bind mount.

Realistically, since we are under `/data` on the host, I assume there is already something that backup the entire `/data` so this script is probably obsoleted.

Reply:
See https://github.com/bird-house/birdhouse-deploy/pull/16/commits/c55725635f18f54a24f0743b31f4546f908a0ba5